Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -63,8 +111,14 @@ public static int[] RemoveAt(int[] array, int index)
         /// <returns>A new array with element added at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] InsertFirst(int[] array, int number)
         {
-            // ToDo: implement.
-            return array;
+            if (array == null || array.Length == 0)
+            {
+                return array;
+            }
+
+            int[] tempArr = new int[array.Length + 1];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'tempArr'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5075368)

Reply:
What would be a good alternative to this variable assignment? The intention is to create a new array that is one larger then the on being added to.

Reply:
Remove this line completely and put int[] in front of the next line. Same goes for the other remarks. Think about why that works and why your line here is doing nothing. :)